docs/concepts/scheduled-jobs.md

```diff
@@ -69,6 +69,14 @@ You can also use a context path to load the project from a local directory conta
 
 Example for a remote GitHub project - https://github.com/mlrun/project-demo
 
+```{admonition} Note
+From MLRun v1.7.1: when running a remote/scheduled workflow, the remote workflow pulls/extracts the remote source content to the running pod but loads the project configuration from the MLRun DB and not from the project.yaml file in the remote source.
+
+The remote files are primarily retrieved for - 
+1. The [project_setup](https://docs.mlrun.org/en/stable/projects/project-setup.html) that may affect the project configuration (if it exists).
```

Comment:
correct link is:  [project_setup](../projects/project-setup.md)

Reply:
Thanks i change the link to the correct one